dataservice/api/participant/models.py

```diff
@@ -27,3 +27,36 @@ class Participant(db.Model, Base):
     demographic = db.relationship(Demographic, backref='participant',
                                   uselist=False, cascade="all, delete-orphan",
                                   lazy=True)
+
+    def upstream_relatives(self):
```

Comment:
I really wasn't sure how to name things here. I thought about having incoming edges to a participant be ancestors and then outgoing edges from a participant be descendants but then decided to keep things more general since we want to represent many types of familial relationships (i.e. father, mother, sister, cousin etc). 

Open to suggestions here!

Reply:
No, not exactly renaming here. `incoming_family_relationships` are incoming edges
to a participant node in the graph. An upstream relative is just the starting
node (or left) node on the incoming edge. And no this wasn't meant to be all
upstream relatives of the participant. I'm going to rename to `upstream_immediate_relatives` and `downstream_immediate_relatives`. And then maybe rename relatives to `immediate_relatives` or something like that.

And yes, I could have a method in the future that gets all relatives of a
participant as well.